{
 "errors": {
  "laspytest": {
   "exception": "Ran out of input",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/parso/cache.py\", line 122, in load_module",
    "    module_cache_item = parser_cache[hashed_grammar][file_io.path]",
    "KeyError: PosixPath('/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/third_party/typeshed/stdlib/2and3/builtins.pyi')",
    "",
    "During handling of the above exception, another exception occurred:",
    "",
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/helpers.py\", line 488, in wrapper",
    "    return func(self, line, column, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 213, in complete",
    "    return completion.complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/completion.py\", line 169, in complete",
    "    cached_name, completion_names = self._complete_python(leaf)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/completion.py\", line 310, in _complete_python",
    "    completion_names += self._complete_global_scope()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/completion.py\", line 375, in _complete_global_scope",
    "    for filter in filters:",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/context.py\", line 495, in get_global_filters",
    "    b = next(base_context.inference_state.builtins_module.get_filters(), None)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/cache.py\", line 44, in wrapper",
    "    rv = function(obj, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/__init__.py\", line 128, in builtins_module",
    "    builtins_module, = self.import_module((module_name,), sys_path=())",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/__init__.py\", line 112, in import_module",
    "    self, import_names, sys_path, prefer_stubs=prefer_stubs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 383, in import_module_by_names",
    "    ) for parent_module_value in value_set",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 383, in <listcomp>",
    "    ) for parent_module_value in value_set",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/plugins/__init__.py\", line 21, in wrapper",
    "    return built_functions[public_name](*args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/plugins/flask.py\", line 20, in wrapper",
    "    return callback(inference_state, import_names, module_context, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/gradual/typeshed.py\", line 127, in wrapper",
    "    parent_module_value, sys_path)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/gradual/typeshed.py\", line 148, in try_to_load_stub_cached",
    "    _try_to_load_stub(inference_state, import_names, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/gradual/typeshed.py\", line 216, in _try_to_load_stub",
    "    m = _load_from_typeshed(inference_state, python_value_set, parent_module_value, import_names)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/gradual/typeshed.py\", line 270, in _load_from_typeshed",
    "    import_names=import_names,",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/gradual/typeshed.py\", line 276, in _try_to_load_stub_from_file",
    "    stub_module_node = parse_stub_module(inference_state, file_io)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/gradual/typeshed.py\", line 293, in parse_stub_module",
    "    use_latest_grammar=True",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/__init__.py\", line 200, in parse",
    "    return self.parse_and_get_code(*args, **kwargs)[0]",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/__init__.py\", line 197, in parse_and_get_code",
    "    return grammar.parse(code=code, path=path, file_io=file_io, **kwargs), code",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/parso/grammar.py\", line 114, in parse",
    "    module_node = load_module(self._hashed, file_io, cache_path=cache_path)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/parso/cache.py\", line 131, in load_module",
    "    cache_path=cache_path",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/parso/cache.py\", line 145, in _load_from_file_system",
    "    module_cache_item = pickle.load(f)",
    "EOFError: Ran out of input",
    ""
   ]
  }
 },
 "symbols": [
  "laspy.__version__",
  "laspy.base",
  "laspy.base.DataProvider",
  "laspy.base.FILE_MODES",
  "laspy.base.FakeMmap",
  "laspy.base.FileManager",
  "laspy.base.HAVE_LAZPERF",
  "laspy.base.Reader",
  "laspy.base.Writer",
  "laspy.base._prepare_np_frombuffer_data",
  "laspy.base.buffer",
  "laspy.base.read_compressed",
  "laspy.base.xrange",
  "laspy.file",
  "laspy.file.File",
  "laspy.file.base",
  "laspy.file.header",
  "laspy.file.util",
  "laspy.glviewer",
  "laspy.glviewer.VBO_Provider",
  "laspy.glviewer.pcl_image",
  "laspy.glviewer.run_glviewer",
  "laspy.header",
  "laspy.header.EVLR",
  "laspy.header.ExtraBytesStruct",
  "laspy.header.Header",
  "laspy.header.LaspyHeaderException",
  "laspy.header.ParseableVLR",
  "laspy.header.VLR",
  "laspy.header.leap_year",
  "laspy.header.string_type",
  "laspy.header.util",
  "laspy.header.xrange",
  "laspy.tools",
  "laspy.tools.lascopy",
  "laspy.tools.lascopy.lascopy",
  "laspy.tools.lascopy.main",
  "laspy.tools.lasexplorer",
  "laspy.tools.lasexplorer.lasexplorer",
  "laspy.tools.lasexplorer.main",
  "laspy.tools.lasnoise",
  "laspy.tools.lasnoise.main",
  "laspy.tools.lasvalidate",
  "laspy.tools.lasvalidate.main",
  "laspy.tools.lasvalidate.validate",
  "laspy.tools.lasverify",
  "laspy.tools.lasverify.lasverify",
  "laspy.tools.lasverify.main",
  "laspy.tools.lasverify.xrange",
  "laspy.tools.lasviewer",
  "laspy.tools.lasviewer.lasview",
  "laspy.tools.lasviewer.main",
  "laspy.util",
  "laspy.util.Format",
  "laspy.util.LEfmt",
  "laspy.util.LaspyException",
  "laspy.util.Point",
  "laspy.util.Spec",
  "laspy.util.defaults",
  "laspy.util.defaults_test",
  "laspy.util.edim_fmt_dict",
  "laspy.util.fmtLen",
  "laspy.util.npFmt",
  "laspytest.test_laspy.LasHeaderWriterTestCase",
  "laspytest.test_laspy.LasLazReaderTestCase",
  "laspytest.test_laspy.LasReaderTestCase",
  "laspytest.test_laspy.LasV_13TestCase",
  "laspytest.test_laspy.LasV_14TestCase",
  "laspytest.test_laspy.LasWriteModeTestCase",
  "laspytest.test_laspy.LasWriterTestCase",
  "laspytest.test_laspy.LazNoPointMapTestCase",
  "laspytest.test_laspy.LazReaderTestCase",
  "laspytest.test_laspy.flip_bit",
  "laspytest.test_laspy.really_copyfile",
  "laspytest.test_laspy.really_remove",
  "laspytest.test_laspy.test_laspy"
 ]
}